boards/common/esp_common/Makefile.dep

```diff
@@ -0,0 +1,7 @@
+# enable bootloader reset over USB, requires CDC ACM to be used
```

Comment:
Makes it really sense to introduce a `esp_common` directory for this single dependency?

Reply:
I'm not sure where else to put it

Reply:
In `cpu/esp32/Makefile.dep`? All boards that allow to use `stdio_cdc_acm` or `tinyusb_class_cdc` can also be reset. The feature is defined at board level but the reset function seems to be a function of the CPU.